src/knockout-jqAutocomplete.js

```diff
@@ -94,9 +94,16 @@
         //the binding's update function. keep value in sync with model
         this.update = function(element, valueAccessor) {
             var options = unwrap(valueAccessor()),
-                value = unwrap(options && options.value);
+                value = unwrap(options && options.value),
+                inputProp = options.inputProp || options.labelProp || options.valueProp,
```

Comment:
Would like to keep the logic for property names in `getPropertyNames` (would be `self.getPropertyNames` here). To avoid unwrapping twice. We could pass options into `getPropertyNames` and then there check if the first arg is a function or not and avoid effectively unwrapping options twice.